CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Lists below only mention user-visible changes, but the full list of contributors
 for this release also includes TK
 
 ### Added
+- Ability to sort feedlist by latest unread (#2492) (Daniel Lublin)
```

Comment:
In general, we leave the changelog writing until release.
@Minoru Would it help you if PRs would include an update to the changelog or do you prefer writing them as part of the release process?

Reply:
I removed any changelog update by this pr